services/line-stat-getter.js

```diff
@@ -82,6 +101,14 @@ function LineStatGetter(model, params, opts) {
     }
   }
 
+  function getGroupByDateFormat() {
+    if (timeRange === 'week' && Database.isSQLite(opts)) {
+      return [opts.sequelize.literal('"YYYY-ww"'), 'format'];
+    } else {
+      return [opts.sequelize.literal('"YYYY-MM-DD"'), 'format'];
+    }
+  }
```

Comment:
I don't really understand why this method is necessary.

Reply:
Why is it passed in the SQL query? It is the same value for each resulting rows, right?

Reply:
I've added a comment to explain you why. Let me know if it's clear now.

Reply:
Why is it passed in the SQL query If it is the same value for each resulting rows?